about.html

```diff
@@ -12,62 +13,90 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
 
 
+  <div class="container about-page">
+
+    <header>
+      <img src="img/lambda-black.png" />
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="#">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+
+    <section>
+      <img class="jumbo" src="img/jumbo-about.png" />
+    </section>
+
+
+
+    <section class="info-section">
+      <h2>About</h2>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate
+        strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world
+        view of disruptive innovation via workplace diversity and empowerment.
+      </p>
+    </section>
+
```

Comment:
Good work using semantic HTML sections and organizing all the sections in their own divs